### DISCUSSION TITLE
Suggestion : ajouter URL dans type de champs additionnels

### DISCUSSION THREAD

**Discussion by sylv1g** [JeromeDevome/GRR#158] [Idées]
L'idée serait de pouvoir ajouter un lien, qui quand on regarderai la réservation (en étant loggé) permettrai d'avoir un lien cliquable.

Aujourd'hui, si je rajoute un champ texte, je ne peux pas le cliquer ni même le sélectionner à moins de faire modifier ou copier cette réservation.

**Comment by ynaessens** [comment]
Bonjour,
même avec un lien cliquable, vu que c'est l'ensemble de la cellule qui est attachée à l'ancre, cela ne fonctionne pas davantage.
Il faudrait mettre à part les champs additionnels du type lien externe, c'est-à-dire revoir la façon dont on affiche les cellules...
Je ne dis pas non mais il y a encore du travail sur la planche.
Cordialement,
YN

**Comment by sylv1g** [comment]
Ça pourrait être un bouton cliquable, comme ceux qui permettent l'édition ?
